hazelcast/src/main/java/com/hazelcast/util/scheduler/EntryTaskScheduler.java

```diff
@@ -47,5 +48,14 @@
 
     void cancelAll();
 
+    /**
```

Comment:
Why do you need to access those scheduled entries? I'm trying to understand how they are being used.